WordPress/src/main/java/org/wordpress/android/GCMRegistrationIntentService.java

```diff
@@ -63,4 +66,22 @@ public void sendRegistrationToken(String gcmToken) {
             AppLog.w(T.NOTIFS, "Empty GCM token, can't register the id on remote services");
         }
     }
+
+
+    public static String getCurrentToken() {
```

Comment:
If it's not used, we should drop that function. `instanceID. getToken ` is asynchronous and we don't want to us outside that Service.